src/main/java/net/fabricmc/filament/task/JavadocLintTask.java

```diff
@@ -0,0 +1,112 @@
+package net.fabricmc.filament.task;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import cuchaz.enigma.ProgressListener;
+import cuchaz.enigma.translation.mapping.EntryMapping;
+import cuchaz.enigma.translation.mapping.serde.MappingFileNameFormat;
+import cuchaz.enigma.translation.mapping.serde.MappingParseException;
+import cuchaz.enigma.translation.mapping.serde.MappingSaveParameters;
+import cuchaz.enigma.translation.mapping.serde.enigma.EnigmaMappingsReader;
+import cuchaz.enigma.translation.mapping.tree.EntryTree;
+import cuchaz.enigma.translation.representation.entry.Entry;
+import cuchaz.enigma.translation.representation.entry.LocalVariableEntry;
+import cuchaz.enigma.translation.representation.entry.MethodEntry;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.file.DirectoryProperty;
+import org.gradle.api.tasks.InputDirectory;
+import org.gradle.api.tasks.TaskAction;
+
+public class JavadocLintTask extends DefaultTask {
+	private static final Pattern PARAM_DOC_START = Pattern.compile("^@param\\s+");
+	private static final Pattern TYPE_PARAM_DOC_START = Pattern.compile("^@param\\s+<");
```

Comment:
Can't the two patterns be merged?

Reply:
Actually, yeah. I was using strings before and just did a simple replacement to regex :smile: